homeassistant/components/otbr/websocket_api.py

```diff
@@ -148,6 +166,21 @@ async def websocket_set_network(
         connection.send_error(msg["id"], "not_loaded", "No OTBR API loaded")
         return
 
+    data: OTBRData = hass.data[DOMAIN]
+
+    try:
+        border_agent_id = await data.get_border_agent_id()
```

Comment:
Maybe make a decorator for getting the OTBRData and another decorator for getting the border agent id?

Examples:
https://github.com/home-assistant/core/blob/3761d1391507835118ff37b8d847a32f0bd6fc77/homeassistant/components/zwave_js/api.py#L301-L363